_posts/2016/2016-11-28-xp-days-germany-2016.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "XP Days Germany 2016"
+date: "2016-11-28 12:11:00"
+image: blog-header/hamburg-harbor-city.jpg
+categories: events
+authors: ["Benjamin", "Bastian"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+The [13th XP Days Germany](http://www.xpdays.de/2016/) took place again at the [Handwerkskammer Hamburg](https://www.hwk-hamburg.de) – within a stone's throw of [ePages HQ](http://www.epages.com).
```

Comment:
"The 13th XP Days Germany took again place at the..." 
the order of the words must be adjusted